packages/esm-patient-labs-app/src/config-schema.ts

```diff
@@ -41,16 +41,53 @@ export const configSchema = {
       _default: '52a447d3-a64a-11e3-9aeb-50e549534c5e',
     },
   },
+  labTests: {
+    _type: Type.Array,
+    _elements: {
+      labTestUuid: {
+        _type: Type.UUID,
+        _description: "UUID for the 'Lab' Test that require order reason",
+        _default: '',
+      },
+      labTestOrderReasons: {
+        _type: Type.Array,
+        _elements: {
+          uuid: {
+            _type: Type.UUID,
+            _description: 'Lab test concept UUID',
+          },
+          label: {
```

Comment:
As a general rule, we should avoid labels in configuration if we can. Labels like this are only usable in monolingual environments. In general, especially if this is concept-backed, we should prefer to use the display strings from the concepts themselves rather than inventing new text for them.

That said, what we store in the backend is just text, so I'm a little unclear on why we even have concepts involved in this at all? If we want to capture coded answers or coded or free text answers, we should actually update the backend and data model to reflect that.

Reply:
At a minimum, labels should be run through the translation system.

Reply:
@ibacher orders model already provides the ability to capture order reason as coded or non coded/ free text. In this case we are using coded answers which are represented by concepts

Reply:
@makombe Thanks. I didn't realise it was using that pattern. If we're using concepts, then the label should come from the concept itself, right? Ideally, we want whatever we're using for drop-down labels to be translatable. Having strings in the configuration schema is, unfortunately, not translatable.

Reply:
Agree with @ibacher here, we should be getting labels from the concepts, right?

Reply:
Then we can simplify the config schema to be just an array of string concept UUIDs. Another alternative, which could be preferable (but I don't feel strongly about it, I'll leave it up to you) would be just to have `orderReasonsConceptSet`, which would be a UUID of a concept set that has the reasons as members.

Reply:
While this would be a good approach, it forces one to have concepts whose displays read exactly as the reasons provided in the drop-down. For instance, there could be a concept for indicating a baseline test with a display of **_initial_** while on the ground folks use **_baseline_** a lot. What would be the workaround for such? Do we support some degree of overrides?

Reply:
Are you not able to change names on your concepts?

Reply:
@brandones we normally try to avoid editing of concept directly as to be in sync with CIEL dictionary. We consume concepts as they come from CIEL.

Reply:
You can always request that CIEL adds a synonym or even preferred name for a concept. Alternatively, you can use Iniz to provide custom concept names or change the preferred name in a reproducible way without needing to deviate from CIEL.

In any case, labels _must never_ appear in the configuration because it is unworkable in a multilingual environment. If we need something in the configuration to represent a user-facing string it needs to be a translation key rather than a label.

Reply:
Great points, thanks. I know adding synonyms in CIEL may be good but still not sustainable more so if preferences/synonyms differ across implementations. A major challenge we face with labels is trying to match the labels as they are provided in clinical forms or even reporting tools. Even though the concept is the same, we are forced to display it differently depending on the type of user, and how they commonly use the concept. 

The concept to use the iniz module is great but I don't see how it will address the label overrides for concepts that may require 2 or 3 synonyms. I may be wrong, but I was hoping that the idea behind a config is to somehow address a local context. For instance, I don't expect the same config to apply in Kenya and Tanzania where English and Kiswahili are used. I would expect the Kenyanized config to be in English and that for Tanzania in Kiswahili. Sorry if I may have missed the point.

Reply:
@ibacher  @brandones I have removed the label from the config as suggested and made necessary adjustment. Thanks

Reply:
Thanks @makombe. @ibacher @brandones @denniskigen kindly help with review. Thanks